tdrs-backend/tdpservice/email/templates/access-request-submitted.html

```diff
@@ -0,0 +1,17 @@
+{% extends 'base.html' %}
+{% block content %}
+<!-- Body copy -->
+<p style="color: #000000;">
+    <p>Dear {{ first_name }},</p>
+
+    <p><b>Your request for {{ stt_name }} access to the TANF Data Portal (TDP) has been submitted for review.</b></p>
+
+    <p>You can <a href="https://tdp-frontend-a11y.app.cloud.gov/" target="_blank">view the status</a> of your request by logging into TDP.</p>
```

Comment:
@n0remac @jtimpe the TDP icon is pointing users to the a11y frontend site. we'd want it to point to prod frontend for real users.  do you want to hard code this to prod or have conditional logic based on the environment (similar to what was done with the [govbanner](https://github.com/raft-tech/TANF-app/pull/1907))?

Reply:
Good catch! @n0remac i think we should update them to use `{% url 'home' %}` or something similar in the templates
https://stackoverflow.com/a/22265090